Dockerfile

```diff
@@ -32,27 +29,29 @@ RUN set -ex \
         Mopidy-GMusic \
         Mopidy-YouTube \
         pyasn1==0.1.8 \
-    # Install dumb-init
-    # https://github.com/Yelp/dumb-init
- && DUMP_INIT_URI=$(curl -L https://github.com/Yelp/dumb-init/releases/latest | grep -Po '(?<=href=")[^"]+_amd64(?=")') \
- && curl -Lo /usr/local/bin/dumb-init "https://github.com/$DUMP_INIT_URI" \
- && chmod +x /usr/local/bin/dumb-init \
     # Clean-up
  && apt-get purge --auto-remove -y \
         curl \
         gcc \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* ~/.cache \
-    # Limited access rights.
- && chown mopidy:audio -R /var/lib/mopidy/.config \
- && chown mopidy:audio /entrypoint.sh
+ # Create the user
+ && useradd -m -d /home/${UNAME} ${UNAME} \
+ # Create the config dir
+ && mkdir -p /home/${UNAME}/.config/mopidy/ \
+ && chown -R ${UNAME} /home/${UNAME}/
+
+# Copy the pulse-client configuratrion
+COPY pulse-client.conf /etc/pulse/client.conf
```

Comment:
Unfortunately there is no way to do that, because the destinations of the files are different.

More info: https://docs.docker.com/engine/reference/builder/#copy
`COPY ["<src>",... "<dest>"]`